scripts-available/CDB_SyncTable.sql

```diff
@@ -88,7 +101,8 @@ DECLARE
   fq_dest_table TEXT;
 
   colnames TEXT[];
-  quoted_colnames TEXT;
+  dst_colnames TEXT;
+  src_colnames TEXT;
 
   src_hash_table_name NAME;
   dst_hash_table_name NAME;
```

Comment:
these two `src_hash_table_name` and `dst_hash_table_name` can be deleted.

Reply:
true